src/Maestro/Maestro.DataProviders/MaestroBarClient.cs

```diff
@@ -150,10 +150,31 @@ private Subscription ToClientModelSubscription(Maestro.Data.Models.Subscription
                 other.TargetBranch)
                 {
                     Channel = ToClientModelChannel(other.Channel),
-                    Policy = ToClientModelSubscriptionPolicy(other.PolicyObject)
+                    Policy = ToClientModelSubscriptionPolicy(other.PolicyObject),
+                    LastAppliedBuild = ToClientModelBuild(other.LastAppliedBuild)
```

Comment:
Is it possible for other.LastAppliedBuild to be null for a newly created or never used subscription?

Reply:
This was fixed